### DISCUSSION TITLE
Bforartists for Appimage instructions

### DISCUSSION THREAD

**Discussion by Draise14** [Bforartists/Bforartists#2569] [General]
**Link here:**
[https://github.com/AppImageCrafters/bforartists-appimage](https://github.com/AppImageCrafters/bforartists-appimage)

> Recipe for packaging Bforartists using the AppImage format.
> The AppImage can be downloaded from the releases assets.
> 
> If you find an issue plese fill a bug.

**Comment by ReinerBforartists** [comment]
Nice find, thanks. Haven't known that there is a finished appimage version around. In the thread azubieta mentioned that he was not able to build it. But seems that he succeeded. 

https://github.com/Bforartists/Bforartists/issues/256

Question remains if the recipe is still valid. And how to build the appimage, which is still not clear described. Somebody wants to build an actual appimage version? :)

**Comment by Draise14** [comment]
Is this recipe still valid?

**Comment by ReinerBforartists** [comment]
Yes

But you have to prepare your environment first. The recipe is just half of the show. I have unfortunately long forgotten the steps. The instructions should be available at the appimage page.
